content/docs/command-reference/push.md

```diff
@@ -33,14 +33,12 @@ use cases for these commands.
 > Those should be uploaded with `git push`.
 
 The default remote is used (see `dvc remote default`) unless the `--remote`
-option is used. See `dvc remote` for more information on how to configure a
-remote.
+option is used. See `dvc remote` for more information on how to configure them.
 
-Without arguments, it uploads all files and directories missing from remote
-storage, found as <abbr>outputs</abbr> of the
-[stages](/doc/command-reference/run) or `.dvc` files present in the workspace.
-The `--all-branches`, `--all-tags`, and `--all-commits` options enable pushing
-multiple Git commits.
+Without arguments, it uploads all files and directories referenced in the
+current workspace (found in `dvc.yaml` and `.dvc` files) that are missing from
```

Comment:
it sounds like _all_ files and directories, which is not true

Reply:
Updated to "uploads the files and directories referenced in the
current workspace (found in all `dvc.yaml`..."